### ISSUE TITLE
image.gaussian not working for torch.ByteTensor

### ISSUE THREAD

**Issue by jagapiou** [torch/image#119]
```lua
image.gaussian{tensor = torch.FloatTensor(1,1)}
```
Works fine.

```lua
image.gaussian{tensor = torch.ByteTensor(1, 1)}
```
Results in a floating point exception.

**Comment by jagapiou** [comment]
Currently working on this https://github.com/torch/image/pull/121.

**Comment by deltheil** [comment]
Solved by #135.
